src/buildVariables.js

```diff
@@ -199,6 +206,9 @@ const buildUpdateVariables = (introspectionResults) => (
 
     // TODO: To overcome this permission issue,
```

Comment:
We can remove the TODO part and just keep the reference to what the upcoming line does for future reference.